app/models/concerns/user_otp_methods.rb

```diff
@@ -21,7 +21,11 @@ def verify_and_enable_otp!(seed, level, otp, expiry)
   def enable_otp!(seed, level)
     self.mfa_level = level
```

Comment:
It should not overwrite mfa_level if webauthn exists already (cause level would already be set). Need to add some logic here to check for that.